jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/MenuSection.java

```diff
@@ -0,0 +1,26 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.common.NestedDropdownMenu;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+
+public class MenuSection {
+
+    @UI("#basic_menu_button")
```

Comment:
Could you please replace underscores with dashes in ids.

Reply:
Fixed, thanks.